final/client/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
```

Comment:
review with Daniel

Reply:
do we even need to generate a dist folder? Running the app converts it on the fly it seems?